media-libs/flac/flac-1.5.0.ebuild

```diff
@@ -41,6 +41,7 @@ multilib_src_configure() {
 	append-flags $(test-flags-CC -fno-ipa-pta)
 
 	local myeconfargs=(
+		# leave enabling multithreading up to the build system
```

Comment:
I don't see what this is commenting on, or the point of it really? It would need to be more specific to be useful..